release-notes/con-relnotes-techpreview-features.adoc

```diff
@@ -10,6 +10,10 @@ These features are for Technology Preview only. Technology Preview features are
 For more information on Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Scope].
 ====
 
+== {product} Operator
+
+You can use the {product} Operator to install {product-short} on your OpenShift Container Platform cluster. For more information, see the link:{LinkAdminGuide}#proc-install-rhdh-operator[Installing {product} using the Operator] section in the _Administration guide for {product}_.
+
 == Plugins available in {product}
```

Comment:
This list doesn't look good. 
As far as I know nothing really changed in the existing plugins support levels. 
The only change is that in 1.1 there is about 10 new plugins (2 supported, 8 as tech preview, see https://docs.google.com/spreadsheets/d/1dNUAGTeosEzreX9fxFTW1Vq3oGCbHroZ_DFEL8sjfjY/edit#gid=0 row 81 and down)

Reply:
see https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/75/files#r1525362220

Reply:
rather than having the list of plugins in a **FOURTH** place, maybe we could just link to the main table in ref-rhdh-plugins.adoc ? https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/75/files#diff-48d14956b347986734708cd4e5df54eadbc40ea3c686e2b1b2e0088fb3f2a655

Reply:
I agree. Provided a cross-reference to that table. 
To get a comprehensive list of supported dynamic plugins, see the link:{LinkAdminGuide}#rhdh-supported-plugins[Dynamic plugins included in {product}] section in the _Administration guide_.